source/documentation/deploying_apps/production_checklist.md

```diff
@@ -6,3 +6,4 @@ Before deploying an app for production use, check the following:
 1. You have [a domain name](https://www.gov.uk/service-manual/technology/get-a-domain-name) for your service.
 1. You are running at least two instances of the app to ensure availability. Use ``cf scale APPNAME -i 2`` or amend the manifest file to add an extra running instance; see [Scaling](/#scaling) for more information.
 1. You are prepared to use a [blue-green deployment process](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) [external link] for when the app needs to be updated or restarted (this avoids problems due to a known issue with the PaaS that can generate transient 404 errors).
+1. You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] of `health-check-type` `http` and set `health-check-http-endpoint` to an appropriate endpoint for your app.
```

Comment:
I understand the content, however I would recommend phrasing it slightly differently:

"You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] by setting `health-check-type` to `http` and setting `health-check-http-endpoint` to an appropriate endpoint for your app."

@samcrang what do you think?

Reply:
Sounds good. I'll tweak it.